### DISCUSSION TITLE
CPU 和 GPU - 异构计算的演进与发展 - 面向信仰编程 · /heterogeneous-computing

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#645] [Announcements]
https://draveness.me/heterogeneous-computing/

**Comment by egege** [comment]
没想到作者对硬件的研究也是这么深入，厉害了。请问大大，能大概讲一下AMD最近的zen架构与rdna架构吗？

**Comment by draveness** [comment]
> @egege
> 没想到作者对硬件的研究也是这么深入，厉害了。请问大大，能大概讲一下AMD最近的zen架构与rdna架构吗？

这块确实没啥研究，之后有机会可以看看

**Comment by aquarhead** [comment]
学习了, 才知道原来 CPU 内部也开始采用 mesh 架构了.

**Comment by Dream4ever** [comment]
1. “哪怕是二级缓存也有将近 30 倍的提升”，7ns 和 100ns 相比，应该是 15 倍左右的提升吧？
2. “那么它会花费大量的事件等待读写主存中的数据” → 时间

---

*2021-04-14 UPDATES: 已修复*
